react-app/src/components/Extractor.js

```diff
@@ -115,181 +129,168 @@ function Extractor(props) {
+                </Button>
+              }
+            />
+          </div>
+        );
+      case 'url':
+        return (
+          <Form.Group className="mb-3" controlId={arg.key} key={arg.key}>
+            <Form.Label>{arg.label}</Form.Label>
+            <div className="file-button-container">
+              <Button
+                onClick={() => {
+                  const newArgs = [...constructorArgsMetadata];
+                  const urlArg = getConstructorArg(newArgs, 'url');
+                  urlArg.hidden = true;
+                  const fileArg = getConstructorArg(newArgs, 'filePath');
+                  fileArg.hidden = false;
+                  updateConstructorArgsMetadata(newArgs);
+                }}
+                className="generic-button narrow-button"
+                variant="outline-info"
+              >
+                Switch to CSV File
+              </Button>
+              <Form.Control
+                type="text"
+                value={arg[arg.key]}
+                onChange={(e) => {
+                  const newArgs = [...constructorArgsMetadata];
+                  const currentArg = getConstructorArg(newArgs, arg.key);
+                  currentArg[arg.key] = e.target.value;
+                  updateConstructorArgsMetadata(newArgs);
+                }}
+                placeholder="Enter URL"
+              />
+            </div>
+          </Form.Group>
+        );
+      case 'dropdown':
+        return (
+          <Form.Group className="mb-3" controlId={arg.key} key={arg.key}>
+            <div className="label-and-icon-container">
+              <Form.Label>{arg.label}</Form.Label>
+              <Form.Text>Select fields to be masked in the extracted Patient resource</Form.Text>
+              <Trash2
+                onClick={() => {
+                  const newArgs = [...constructorArgsMetadata];
+                  const currentArg = getConstructorArg(newArgs, arg.key);
+                  currentArg.hidden = true;
+                  updateConstructorArgsMetadata(newArgs);
+                }}
+                className="mouse-pointer"
+              />
+            </div>
+            {arg.options.map((option) => (
+              <Form.Check
+                label={option}
+                name={option}
+                type="checkbox"
+                value={option}
+                key={`mask-${option}`}
+                id={`mask-${option}`}
+                className="arg-input-width-limit"
+                onChange={(e) => {
+                  const newArgs = [...constructorArgsMetadata];
+                  const currentArg = getConstructorArg(newArgs, arg.key);
+                  updateOptions(currentArg, e.target.value, arg.key);
+                  updateConstructorArgsMetadata(newArgs);
+                }}
+              />
+            ))}
+          </Form.Group>
+        );
+      default:
+        console.error('Unexpected ConstructorArg type: ', arg.type);
```

Comment:
How do you feel adding a console statement rule to eslint so this doesn't complain?